manifests/mongodb_client.pp

```diff
@@ -0,0 +1,67 @@
+# Certs configurations for MongoDB
+class certs::mongodb_client (
+  $hostname             = $certs::node_fqdn,
+  $cname                = $certs::cname,
+  $generate             = $certs::generate,
+  $regenerate           = $certs::regenerate,
+  $deploy               = $certs::deploy,
+  $pki_dir              = '/etc/pulp/mongodb',
+  $server_cert          = $certs::server_cert,
+  $country              = $certs::country,
+  $state                = $certs::state,
+  $city                 = $certs::city,
+  $org                  = $certs::org,
+  $org_unit             = $certs::org_unit,
+  $expiration           = $certs::expiration,
+  $default_ca           = $certs::default_ca,
+  $ca_key_password_file = $certs::ca_key_password_file,
+  $group                = $certs::group,
```

Comment:
Should this be
```suggestion
  $group                = 'pulp',
```

And then use that instead of `$pulp_user`? The variable `$group` is not used elsewhere.